deploy/chart/catalog_resources/certified-operators/mongodb-enterprise.v0.3.2.clusterserviceversion.yaml

```diff
@@ -1,263 +1,48 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
+  name: mongodboperator.v0.3.2
+  namespace: placeholder
+
```

Comment:
Is the extent of your changes just adding whitespace?

Reply:
I've updated the description of this PR with the relevant changes.